### PR TITLE
Fill most management UI info

### PR DESCRIPTION
**Pull request by essen**
Part of https://github.com/rabbitmq/rabbitmq-stomp/issues/55

### CODE REVIEW

**Comment by essen** [comment]
I'll rebase, and I'll add another fix.

File: src/rabbit_stomp_processor.erl [hunk 1]
```diff
@@ -110,7 +110,16 @@ flush_and_die(State) ->
...
 initial_state(Configuration, 
-    {SendFun, ReceiveFun, AdapterInfo, StartHeartbeatFun, SSLLoginName, PeerAddr}) ->
+    {SendFun, ReceiveFun, AdapterInfo0 = #amqp_adapter_info{additional_info=Extra},
+     StartHeartbeatFun, SSLLoginName, PeerAddr}) ->
+  %% STOMP connections use exactly one channel. The frame max is not
+  %% applicable and there is no way to know what client is used.
+  AdapterInfo = AdapterInfo0#amqp_adapter_info{additional_info=[
+       {channels, 1},
+       {channel_max, 1},
+       {frame_max, 'N/A'},
```
Comment (michaelklishin) [inline src/rabbit_stomp_processor.erl]: I think `131072` is the default we have in most clients. Should we use that? a string `frame_max` is not expected by any part of the codebase except for the UI, I think.

Comment (essen) [inline src/rabbit_stomp_processor.erl]: It has no direct meaning for a protocol like STOMP though, I thought? As far as I understand the protocol is HTTP-like and the limits mentioned in the spec are in number of headers, size of header lines and size of body.

Comment (michaelklishin) [inline src/rabbit_stomp_processor.erl]: Fair enough. Then maybe it should be `undefined` or `null`?

Comment (essen) [inline src/rabbit_stomp_processor.erl]: If it's an atom it's printed as-is in the UI. I am open to any value there, I'm not sure which one is clearer for users.

**Comment by essen** [comment]
Rebased and with extra fix to avoid an annoying crash log when client disconnects brutally.

File: src/rabbit_stomp_processor.erl (hunk 1 above)
Comment (michaelklishin) [inline src/rabbit_stomp_processor.erl]: @essen  then `'N/A' is reasonable. I'll check if this affects any HTTP API clients we test against.

Comment (michaelklishin) [inline src/rabbit_stomp_processor.erl]: ’N/A’ is displayed as ’N/A bytes’ in the UI and it does break HTTP API clients in statically typed languages (Go, Java). I think we should use `0` then.

**Pull request merged**